webpack/components/EditableTextInput/EditableTextInput.js

```diff
@@ -140,7 +132,7 @@ const EditableTextInput = ({
         </SplitItem>
         { isPassword ?
           <SplitItem>
-            <Button aria-label={`show-password ${attribute}`} variant="plain" isDisabled={inputValue?.length < 1} onClick={toggleShowPassword}>
+            <Button aria-label={`show-password ${attribute}`} variant="plain" isDisabled={!inputValue?.length > 0} onClick={toggleShowPassword}>
```

Comment:
```js
isDisabled={!inputValue?.length > 0}
```
Do you mean `!(inputValue.length > 0)` or `!(inputValue.length) > 0`? I think without the parenthesis you're comparing `true` and `0`, which works (thanks JavaScript) but is too confusing for me.

Either
```js
!(inputValue?.length > 0)
```
or simply
```js
!inputValue.length
```
(since `0` is falsey in JS)
would make more sense to me.